coda/coda_mdstore/factories.py

```diff
@@ -66,6 +66,50 @@ class FullBagFactory(BagFactory):
     )
 
 
+class OAIBagFactory(BagFactory):
+    """
+    Factory to create a Bag object with two related Bag_Info objects, and
```

Comment:
Seems like this is actually creating 4 related Bag_Info object. Also, both numbers should be either the numerical glyph or the spelled out number.